model/testing/src/icon4py/model/testing/pytest_config.py

```diff
@@ -163,3 +163,12 @@ def pytest_runtest_setup(item):
         backend,
         is_datatest=item.config.getoption("--datatest"),
     )
+
+
+# Replace "fullname" of pytest benchmarks with a shorter name for better readability in bencher
+def pytest_benchmark_update_json(output_json):
+    for bench in output_json["benchmarks"]:
+        # Store the original fullname
+        bench["fullname_original"] = bench["fullname"]
```

Comment:
where is this used?

Reply:
(Replying here since I've added this change)
`fullname_original` is not used anywhere at the moment. I just kept it in case this hook needs to be extended in the future or there is some need to read the original `fullname` by another tool that processes the generated `.json` file. It can be removed.

Reply:
The original `fullname` also includes the file where the test was found but I agree that probably it won't ever be necessary